en/docs/install-and-setup/setup/security/enabling-hostname-verification.md

```diff
@@ -18,7 +18,7 @@ The hostname verification is **disabled** by default. This is done using the `o
 
 !!! note "Strict hostname verification"
 
-    When applying the strict hostname verification, you need to change the configuration in the `site/public/conf/settings.js` file of the Publisher and Admin Portal and the `site/public/theme/setting.js` file of the Developer Portal.
+    When applying the strict hostname verification, you need to change the configuration in the `<APIM_HOME>/repository/deployment/server/webapps/publisher/site/public/conf/settings.json` file of the Publisher and Admin Portal and the `site/public/theme/setting.js` file of the Developer Portal.
```

Comment:
Let's add the relevant full path to devportal settings file as well.

Reply:
Addressed via https://github.com/wso2/docs-apim/pull/7055/commits/1a5991c978624e7b60f1941706cbf0362bde6f59

Reply:
As this is referring to the path in the Devportal app, it has to be `<APIM_HOME>/repository/deployment/server/webapps/devportal/site/public/theme/settings.json`. Shall we correct it Ruthryi?

Reply:
Corrected with https://github.com/wso2/docs-apim/pull/7055/commits/6a21267d961d6df499fbb48e70b0c994a0de345d